ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -7,6 +7,206 @@
+      regexKo: Regexes.ability({ id: '4BA5', source: '라무', capture: false }),
+      delaySeconds: 5,
+      preRun: function(data) {
+        data.firstSpear = data.firstSpear || 1;
+      },
+      infoText: function(data) {
+        if (data.firstSpear == 1) {
+          return {
+            en: 'Look for small spear',
+            ko: '작은 지팡이 확인',
+          };
+        }
+      },
+      run: function(data) {
+        data.firstSpear++;
+      }
+    },
+    {
+      id: 'E5S Tribunal Summons',
+      regex: Regexes.startsUsing({ id: '4BAC', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BAC', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BAC', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BAC', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BAC', source: '라무', capture: false }),
+      preRun: function(data) {
+        data.adds = data.adds || 0;
+      },
+      infoText: function(data) {
+        if (data.adds != 0) {
+          return {
+            en: 'Look for adds',
+            ko: '쫄 위치 확인',
+          };
+        }
+      },
+      run: function(data) {
+        data.adds++;
+      }
+    },
+    {
+      id: 'E5S Fury\'s Bolt',
+      regex: Regexes.startsUsing({ id: '4BAA', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BAA', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BAA', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BAA', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BAA', source: '라무', capture: false }),
+      infoText: {
+        en: 'Fury\'s Bolt',
+        ko: '라무 강화',
+      },
+    },
+    {
+      id: 'E5S Judgment Volts',
+      regex: Regexes.startsUsing({ id: '4BB5', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BB5', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BB5', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BB5', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BB5', source: '라무', capture: false }),
+      infoText: {
+        en: 'aoe',
+        de: 'AoE',
+        fr: 'Dégâts de zone',
+        cn: 'AOE',
+        ko: '전체 공격',
+      },
+    },
+    {
+      id: 'E5S Stepped Leader',
+      regex: Regexes.startsUsing({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BC6', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BC6', source: '라무', capture: false }),
+      preRun: function(data) {
+        data.leaderCount = data.leaderCount || 0;
```

Comment:
My understanding is that Stepped Leader is different based on whether or not Ramuh has a stack from Fury's Bolt on him or not.  I thought it could be random later in the fight, but maybe not?  Mostly I'm wondering if it makes sense to track the stack on Ramuh as being there or not and then using that to determine what to calll for Stepped Leader.

Reply:
ah, yes, you can determine whether Ramuh's Fury something or not. It is determinable by counts but I think your opinion is better. It is not random as far as I played.

Reply:
Ok, if it's not random, then doing it by count sounds fine!  I just wasn't sure.

Reply:
This is fine if this works... 😅 This is not working properly.

> There's something needed to be fixed. E5S Stepped Leader has problem, it only alerts data.leaderCount %2 == 0 (ready spread). I believe this logic should do both. Strangely, E5S Stepped Leader Spread is working well. only when data.leaderCount %2 == 0, it says to Spread.
I don't know why this happens..
I don't know why this is not working...

Reply:
Do you know how to add condition, Ramuh hasEffects id: '8B7'(Fury's Bolt)?
I can't find any examples.